README.md

```diff
@@ -85,6 +85,12 @@ The Dart package workflow consists of the following steps:
 
 **Default** `"lib test"`
 
+#### `check_ignore`
+
+**Optional** Add the flag `--check-ignore` to ignore lines from coverage depending on the comments.
+
+**Default** `""`
```

Comment:
@wolfenrain @renancaraujo I think it is more sensible to have it enabled by default. Since if you are using comments to ignore particular sections of the code you're most likely doing it purposely already.